### PR TITLE
Add dartdoc to typography.dart

### PR DESCRIPTION
**Pull request by abarth**

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

**Comment by abarth** [comment]
TBR @Hixie 

Happy to iterate on these docs.

File: packages/flutter/lib/src/material/typography.dart [hunk 1]
```diff
@@ -69,11 +110,30 @@ class TextTheme {
...
   }
-
 }
 
+/// The two material design text themes.
+///
+/// [Typography.black] and [Typography.white] define the two text themes used in
+/// material design. The black text theme, which uses darkly colored glyphs, is
+/// used on lightly colored backgrounds in light themes. The white text theme,
+/// which uses lightly colored glyphs, is used on darkly colored backgrounds in
```
Comment (Hixie) [inline packages/flutter/lib/src/material/typography.dart]: reverse the order here, "used in dark themes and on darkly-colored..."

File: packages/flutter/lib/src/material/typography.dart [hunk 2]
```diff
@@ -69,11 +110,30 @@ class TextTheme {
...
+///
+/// See also:
+///
+///  * [Theme]
+///  * [ThemeData]
+///  * <http://www.google.com/design/spec/style/typography.html>
 class Typography {
   Typography._();
+
+  /// A material design text theme with darkly colored glyphs.
```
Comment (Hixie) [inline packages/flutter/lib/src/material/typography.dart]: darkly-colored.

But "dark" is probably sufficient.

File: packages/flutter/lib/src/material/typography.dart [hunk 3]
```diff
@@ -69,11 +110,30 @@ class TextTheme {
...
   static const TextTheme black = const TextTheme._black();
+
+  /// A material design text theme with lightly colored glyphs.
```
Comment (Hixie) [inline packages/flutter/lib/src/material/typography.dart]: lightly-colored.

But "light" is probably sufficient. In particular, "lightly-colored" means "has a little bit of color" which probably isn't what we mean here.

**Comment by Hixie** [comment]
LGTM

**Pull request merged**